src/approaches/interactive_learning_approach.py

```diff
@@ -152,15 +150,29 @@ def _relearn_predicates_and_nsrts(
 
     def get_interaction_requests(self) -> List[InteractionRequest]:
         requests = []
+        # Create the explorer.
+        explorer = create_explorer(
+            CFG.explorer,
+            self._get_current_predicates(),
+            self._initial_options,
+            self._types,
+            self._action_space,
+            self._train_tasks,
+            self._get_current_nsrts(),
+            self._option_model,
+            babble_predicates=self._predicates_to_learn,
+            atom_score_fn=self._score_atom_set_frequency,
```

Comment:
it's more because we only have one such atom score function implemented right now. note the difference between glib and greedy lookahead score functions: glib can only score atoms (because it babbles an atomic goal, which doesn't have a full state associated with it) whereas the score function used by greedy lookahead can depend on the low-level state (e.g., entropy stuff).